<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>741</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Подбор оптимального состава группы БЛА</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pareto_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>600</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить множество Парето</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>t = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nmin_input">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>min =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nmax_input">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>max =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t1min">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>min =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t2min_input">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>min =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t3min_input">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>min =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_p_input">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>p =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_scalar">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Скалярная свертка критерия:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_a1_input_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>570</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>a  =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_a2_input_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>570</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>a  =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_target">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Метод прицельной точки:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_t">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>300</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_a1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>570</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_a2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>570</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_nmin">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_nmax">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_p">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.7</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_t3min">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_t3max">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>500</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_t1min">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>80</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_t2min">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_t1max">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1200</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_t2max">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_input">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Блок ввода параметров</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t1max">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>max =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t2max_input">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>max =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_t3max_input">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>500</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>max =</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>600</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать критерии</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output_scalar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>271</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output_target">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>271</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_intervals_input">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>560</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество интервалов:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="input_interval_count">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>560</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_m_input">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>m =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_m">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_errormessage">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>670</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="text_t">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Время операции:</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_n">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Имеющееся количество БЛА:</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_m">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Предполагаемое число целей:</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_p_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вероятность успешного</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_p_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>поражения цели:</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_ts_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Границы временных</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_ts_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>306</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>параметров (в минутах)</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_t1_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Среднее время сохранения</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_t1_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>346</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>работоспособности (t ):</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_t1_3">
    <property name="geometry">
     <rect>
      <x>196</x>
      <y>350</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_t2_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Среднее время обнаружения</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_t2_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>416</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>и идентификации (t ):</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_t2_3">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>420</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_t3_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>466</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Средняя продолжительность</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_t3_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>480</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>атаки по цели (t ):</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_t3_3">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>486</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_as_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Весовые коэффициенты</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_as_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>546</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>для скалярной свертки:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_a1_input_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>570</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_a2_input_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>569</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  2</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
